smart_contracts/contract/src/contract_api/system.rs

```diff
@@ -143,6 +143,13 @@ pub fn transfer_to_account(target: AccountHash, amount: U512, id: Option<u64>) -
     TransferredTo::result_from(transferred_to_value)
 }
 
+/// Transfers `amount` of motes from the default purse of the account to `target` public key. If
+/// account referenced by a `target` public key does not exist it will be created.
```

Comment:
```suggestion
/// Transfers `amount` of motes from the main purse of the caller's account to the main purse of
/// `target`.  If the account referenced by `target` does not exist, it will be created.
```